README.md

```diff
@@ -1,29 +1,45 @@
 # palang
+
 Simple authentication service, to help create application faster. Main Idea is help developer not write again auth and user module when create new application.
 
-### Feature
+## Feature
+
 - Register
 - Login
 - Detail User
 - Validate Token
 
-### Foldering
+## Project Structure
+
 Basic idea for foldering we breakdown breakdown to 3 folder, Service, Manager and Delivery
+
 - *Service*
 in this folder contain logic and functions of each domain, and each domain must be single responsibility
 - *Manager*
-in this folder contain flow/usecase for apps (can be orchestration multiple services). example function to Register that will involve user servics and email services. 
+in this folder contain flow/usecase for apps (can be orchestration multiple services). example function to Register that will involve user servics and email services.
 - *Delivery*
 in this it's delivery protocol, so can be multiple delivery (grpc, rest API, Graphql)
 
 ## How To Run
+
+Copy env sample `cat env.sample > .env`
+
+### Run test
+
+`make test`
```

Comment:
NICE!

Reply:
Please help integrate this repo using codecov opensource(I'm not administrator ), the result looks like this PR https://github.com/bukalapak/envsync/pull/11